.github/workflows/docs.yaml

```diff
@@ -0,0 +1,102 @@
+name: Docs
+
+on:
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        type: boolean
+        description: Debug with tmate
+        required: false
+        default: false
+
+jobs:
+  # This has been migrated to CircleCI
+  # test:
```

Comment:
Leaving the `test` job as a reference should we ever need to move back to testing on Github Actions

Average job time here is ~1-2 minutes, but the disadvantage is that we lose access to CircleCI's built-in analytics dashboards